logdetective/server/metric.py

```diff
@@ -17,8 +17,8 @@
 
 async def add_new_metrics(
     api_name: str,
-    url: str,
-    http_session: aiohttp.ClientSession,
+    url: str = None,
+    http_session: aiohttp.ClientSession = None,
```

Comment:
We'll need to change typing hints to `Optional` for this. Otherwise the IDE will complain. Also, the same change should be applied to other arguments.

Reply:
Indeed. It's something we should do, but it's somewhat lower priority than... a lot of stuff. It's just that I wouldn't like to introduce more of these cases.